README.md

```diff
@@ -46,11 +70,6 @@ yarn test:all
 ### Versioning
 
 We are using [lerna](https://lerna.js.org/) as a package versioning tool. It helps with keeping multiple package versions in sync.
-To upgrade version, first commit your changes to feature branch and push it to github. Then run
-```
-yarn new-version
```

Comment:
We're removing this…why?

Reply:
Because I've been just incrementing the version with `lerna publish` when I do the publish, and I think that's easier to manage than having individual engineers try to bump versions and then keep it all in sync. I'm planning to do some maintainer-oriented documentation in another PR, but in the meantime I just wanted to remove a suggestion I didn't want anyone to follow in the short-term.

Reply:
OK, got it.